### PR TITLE
Add Katello 3.17

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/centos.org/jobs/katello-pipelines.yml [hunk 1]
```diff
@@ -16,15 +16,15 @@
       - katello
 
 - project:
-    name: katello-release-3.16
+    name: katello-release-3.16-3.17
```
Comment (pcreech) [inline puppet/modules/jenkins_job_builder/files/centos.org/jobs/katello-pipelines.yml:19]: Um.... Should 3.16 still be here?

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/centos.org/jobs/katello-pipelines.yml:19]: yep, 3.16 and 3.17 have the same job configuration, this is just a naming thing

Comment (pcreech) [reply puppet/modules/jenkins_job_builder/files/centos.org/jobs/katello-pipelines.yml:19]: ah, k.  WFM

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/centos.org/jobs/katello-pipelines.yml:19]: Now that you dropped centos8, there actually isn't a difference between 3.14, 3.15, 3.16 and 3.17, right?

**Pull request merged**